src/components/BootProto.js

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) [2020] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useState, useEffect } from 'react';
+import { Modal, ModalVariant, Button, FormSelect, FormSelectOption, TextInputBase } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody, TableVariant } from '@patternfly/react-table'
+import { useNetworkDispatch } from '../NetworkContext';
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+
+const BOOT_PROTOCOL_MODES = {
+    none:    _("None"),
+    static:  _("Static"),
+    dhcp:    _("DHCP")
+};
+
+const bootProtocolOptions = Object.keys(BOOT_PROTOCOL_MODES).map(key => {
+    return { value: key, label: BOOT_PROTOCOL_MODES[key] };
+});
+
+const linkText = (connection) => {
+  if (!connection) {
+      return _("Not configured") ;
+  }
+
+  if (connection.iP) {
```

Comment:
It should show the IP when it is manually configured or the bootproto otherwise.

Reply:
Ok, I'll change it ASAP.

Just a note, initially I put the IP address + the bootproto inside brackets - i.e.  `192.168.2.8 (DHCP)'  - but I realized that we only know the IP when it is manually set.

That's what the reason I end up with this approach :)

Reply:
> Just a note, initially I put the IP address + the bootproto inside brackets - i.e.  `192.168.2.8 (DHCP)`  - but I realized that we only know the IP when it is manually set.

I did it because the column is called "IP".